server/models/Product.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const productSchema = new mongoose.Schema({
+    name: {
+      type: String,
+      required: true
+    },
+    description: {
+      type: String
+    },
+    url: {
+      type: String,
+      required: true
+    },
+    pictureUrl: {
+      type: String,
+      required: true
+    },
+    price: {
+      type: Number,
+      required: true
+    }
```

Comment:
i think there should be a field that indicates the source of the product; this way we will know which scraping service to call

Reply:
url will be used for it